actions/views/rhs.ts

```diff
@@ -429,8 +431,12 @@ export function showChannelInfo(channelId: string) {
 }
 
 export function closeRightHandSide() {
-    return (dispatch: DispatchFunc) => {
-        dispatch(batchActions([
+    return (dispatch: DispatchFunc, getState: GetStateFunc) => {
+        const state = getState() as GlobalState;
+        const editingPost = getEditingPost(state);
```

Comment:
`editingPost` seems not needed as it will always be the same, I guess we need `getSelectedPostId` to match `editingPostId`.

(see `getIsPostBeingEditedInRHS`)

```ts
export function getIsPostBeingEditedInRHS(state: GlobalState, postId: string) {
    const editingPost = getEditingPost(state);

    return editingPost.isRHS && editingPost.postId === postId && state.views.posts.editingPost.show;
}
```

would it suffice to just use `editingPost.isRHS`?